docs/design/bundle-sources.md

```diff
@@ -0,0 +1,240 @@
+# Building a file-based catalog from a plain bundle image
+
+**IMPORTANT:** Operator Lifecycle Manager (OLM) v1 features and components are still experimental. Early adopters and contributors should expect breaking changes. The following procedures are not recommended for use on production clusters.
```

Comment:
Nit: This might be a good place to add a Github markdown warning.

Could look something like:
> **Warning**: some warning text here ...